### PR TITLE
OAS generation and Dev Spaces

### PR DESCRIPTION
**Pull request by jasonmadigan**
Draft PR

### CODE REVIEW

**Comment by jasonmadigan** [comment]
@david-martin ready for review

**Comment by jasonmadigan** [comment]
Related docs PR: https://github.com/Kuadrant/docs.kuadrant.io/pull/84

File: doc/openapi-openshift-dev-spaces.md [hunk 1]
```diff
@@ -0,0 +1,351 @@
...
+To follow along, you'll need access to a Dev Spaces instance. This can be either:
+
+- A self-hosted instance.
+- An instance provided through the [Red Hat Developer Sandbox](https://developers.redhat.com/developer-sandbox/ide).
+
+## Setting up your Workspace
+
+First, create a Workspace in Dev Spaces for your project:
+
+1. Fork the following repository: `https://github.com/Kuadrant/blank-petstore`.
```
Comment (david-martin) [inline doc/openapi-openshift-dev-spaces.md:17]:
```suggestion
1. Fork the following repository: [https://github.com/Kuadrant/blank-petstore](https://github.com/Kuadrant/blank-petstore).
```

**Review by david-martin — CHANGES_REQUESTED** [review]
Few small suggestions for ease of following,
otherwise, looks good.
Got through it ok

File: doc/openapi-openshift-dev-spaces.md [hunk 2]
```diff
@@ -0,0 +1,351 @@
...
+### Implementing `AuthPolicy` and Security Schemes
+
+To secure the `/user/login` endpoint with API key authentication, use the following configuration:
+
+```yaml
+/user/login:
+  # ...
+  get:
+    security:
+    - api_key: []
```
[the remark this answers is not in the document]
Comment (david-martin) [reply doc/openapi-openshift-dev-spaces.md]: Yeah, exactly.
The securityScheme one was indented and made copy/paste easy.
The others, I had to copy/paste & fix indent. 
Not major, so I'll defer to your judgement on what seems most reasonable

**Comment by jasonmadigan** [comment]
@david-martin fixed up

**Review by david-martin — COMMENTED** [review]
/lgtm
/approve

**Review by david-martin — APPROVED** [review]

**Pull request merged**